api/lib/opentelemetry/version.rb

```diff
@@ -6,5 +6,5 @@
 
 module OpenTelemetry
   ## Current OpenTelemetry version
-  VERSION = '0.12.1'
+  VERSION = '0.12.2'
```

Comment:
I was about to rant about the divergence in SDK and API version numbers, but I re-read the [versioning and stability doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#version-numbers) and this is explicitly allowed.